src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+@use './variables' as *;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: $star-size;
+    width: $star-size;
+    margin-right: $star-margin;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} :nth-child(-n + #{$i}) {
+      background-image: $stars-background;
```

Comment:
Make sure that the `$stars-background` variable is defined in the 'variables.scss' file and contains a valid image URL for the filled star. If the variable is not defined or the URL is incorrect, the filled stars will not display as intended.